### PR TITLE
suggested new intro para

### PR DESCRIPTION
**Pull request by eddelbuettel**

### CODE REVIEW

**Comment by cboettig** [comment]
Whoops, we're both making the same / conflicting edits I think.  I've added this paragraph to the start as well, since I think it's a good big picture / quick start.  I opted not to include any explicit command line example though, since we give examples of these in the later sections with more detail and introducing the syntax there might be a stumbling block?

I dunno, but take a quick look at the current master version and let me know how it reads.

**Comment by eddelbuettel** [comment]
Yup, let's close this and I pivor to master.

**Comment by eddelbuettel** [comment]
But the PR added a bit more in terms of working and fixed a typo (create -> created).

File: manuscript.Rmd [hunk 1]
```diff
@@ -135,7 +153,6 @@ environment (e.g. `R`, though other common options may be `Rscript` or
 
 This example shows the use of the `--rm` flag to indicate that the
 container should be removed when the interactive session is finished.
-
```
Comment (cboettig) [inline manuscript.Rmd:138]: good catch, just pushed this to master.

not seeing the create -> created edit in the PR though?

**Comment by eddelbuettel** [comment]
I did a subsequent round on master addressing that (meaning the two paras): https://github.com/rocker-org/rocker-paper/commit/9629b203622239dd407d3d556d3ceda47e00262a

Thanks for catching the empty line change.

**Pull request closed without merging**